src/main/java/org/jenkinsci/plugins/codesonar/conditions/RedAlertLimitCondition.java

```diff
@@ -1,30 +1,35 @@
 package org.jenkinsci.plugins.codesonar.conditions;
 
-import hudson.Extension;
-import hudson.Launcher;
-import hudson.model.TaskListener;
-import hudson.model.Result;
 import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import javax.annotation.Nonnull;
 
-import hudson.util.FormValidation;
 import org.jenkinsci.Symbol;
-import org.jenkinsci.plugins.codesonar.CodeSonarBuildAction;
+import org.jenkinsci.plugins.codesonar.CodeSonarLogger;
 import org.jenkinsci.plugins.codesonar.models.CodeSonarBuildActionDTO;
 import org.jenkinsci.plugins.codesonar.models.analysis.Alert;
+import org.jenkinsci.plugins.codesonar.models.analysis.Analysis;
 import org.kohsuke.stapler.DataBoundConstructor;
 import org.kohsuke.stapler.DataBoundSetter;
-import hudson.model.Run;
 import org.kohsuke.stapler.QueryParameter;
 
-import javax.annotation.Nonnull;
+import hudson.Extension;
+import hudson.Launcher;
+import hudson.model.Result;
+import hudson.model.TaskListener;
+import hudson.util.FormValidation;
 
 /**
  *
  * @author Andrius
  */
 public class RedAlertLimitCondition extends Condition {
+    private static final Logger LOGGER = Logger.getLogger(RedAlertLimitCondition.class.getName());
 
     private static final String NAME = "Red alerts";
+    private static final String RESULT_DESCRIPTION_MESSAGE_FORMAT = "threshold={0,number,0} count={1,number,0}";
```

Comment:
To make this consistent with other conditions, there should be a comma between "threshold" and "count", e.g. "threshold=1, count=0".